app.core/resources/public/templates/textadept.txt

```diff
@@ -0,0 +1,138 @@
+;;; {{themename}}.lua
+
+;; Copyright (C) {{year}} , {{themeauthor}}
+
+;; Author: {{themeauthor}}
+;; Version: 0.1
+;; Package-Requires: ((emacs "24"))
```

Comment:
This probably does not belong here :)

Reply:
You are right in all... I withdraw this PR.

Reply:
Sorry... too much things going on in parallel. I'm trying to install everything to test on a VM with Ubuntu (server) to test on an external WWW. But the only thing it shows is a that it it trying to launch a WWW server and then nothing else. Installing a text web browser (lynx) takes me to a page that takes me to the Web. Could you please help me getting the environment ready to test before I do my next PR ;-)
Thanks a ton...I really want ThemeCreator a reality for TextAdept.

Reply:
are you using mac or windows ?